posthog/models/filters/properties_timeline_filter.py

```diff
@@ -0,0 +1,14 @@
+from posthog.models.filters.base_filter import BaseFilter
+from posthog.models.filters.mixins.common import DateMixin, EntitiesMixin
+from posthog.models.filters.mixins.groups import GroupsAggregationMixin
+from posthog.models.filters.mixins.property import PropertyMixin
+
+
+class PropertiesTimelineFilter(
```

Comment:
Worth chatting with @mariusandra and making this use data exploration only instead?

Reply:
Someone needs to build this backend router... 

But, keeping the backend as is (with a router in `query.ts`), could the frontend be built on top of data exploration? 🤔